magicctapipe/conftest.py

```diff
@@ -384,6 +386,13 @@ def config_monly():
     config_path = resource_file("config_monly.yaml")
     return config_path
 
+@pytest.fixture(scope="session")
+def config_gen():
+    config_path = resource_file("config_general.yaml")
```

Comment:
it would be better that the config files used in the tests have it clearly stated in the file name, like config_test_general.yaml (we should also change it for the other 2 configs that are already merged)

Reply:
Done. They all start with 'test_' now